src/Microsoft.AspNet.Mvc.Core/Rendering/JsonHelper.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using Microsoft.AspNet.Mvc.Core;
+using Newtonsoft.Json;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.Rendering
+{
+    /// <summary>
+    /// Default implementation of <see cref="IJsonHelper"/>.
+    /// </summary>
+    public class JsonHelper : IJsonHelper, ICanHasViewContext
+    {
+        private ViewContext _viewContext;
+        private JsonOutputFormatter _jsonOutputFormatter;
+
+        /// <inheritdoc />
+        public ViewContext ViewContext
+        {
+            get
+            {
+                if (_viewContext == null)
+                {
+                    throw new InvalidOperationException(Resources.HtmlHelper_NotContextualized);
```

Comment:
need to either merge these methods into `IHtmlHelper` or add a new resource with a message about `JsonHelper`